lib/request_handler/concerns/config_helper.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module RequestHandler
+  module Concerns
+    module ConfigHelper
+      def lookup!(hash, key)
+        hash.dig(*symbolize_key(key)).tap do |data|
+          raise NoConfigAvailableError, key.to_sym => 'is not configured' if data.nil?
+        end
+      end
+
+      def lookup(config, key)
+        config.dig(*symbolize_key(key))
+      end
+
+      def symbolize_key(key)
+        key.split('.').map(&:to_sym)
```

Comment:
hmm would it make sense to keep the result of this in a cache? it's called a lot on every single request

Reply:
mhm... would this really make sense? I think lookup is for most keys only called once.

Reply:
ok maybe I misunderstood some code then, thought it's called on each request and not only during initialization

Reply:
No I think I did ;-) You're right.